frontend/src/components/App/AppSidebar/SettingsBar.jsx

```diff
@@ -39,7 +39,15 @@ class SettingsBar extends PureComponent {
                 cancelButtonText: 'No, dont do it!'
             }).then((result) => {
                 if (!result.dismiss) {
-                    this.props.socket.emit('logout', {data: this.props.token});
+                    fetch(constants.config.url + '/rooms/' + window.location.pathname.split('/')[2] + '/delete' , 
```

Comment:
maybe we could write a fetch wrapper which would automatically prepend `constants.config.url`. 
For more info on how to implement this check https://github.com/Gabsii/sprint-slides-generator/blob/master/utils/api.js

**EDIT:** I saw that fetch was used multiple times like this, please higher the priority of this comment and replace it everywhere possible

Reply:
There also is a much better way to get the roomId (window.location.pathname.split('/')[2] is the roomID I assume). It leverages react hooks (new React API), I will look into migrating our components from class-based components to functional components once this PR is merged. 
Issue #234 will be used for further discussion

Reply:
Fetch wrapper resolved